bootstrap/app.php

```diff
@@ -9,7 +9,7 @@
         \Prologue\Alerts\AlertsServiceProvider::class,
     ])
     ->withRouting(
-        web: __DIR__.'/../routes/web.php',
+        web: __DIR__.'/../routes/base.php',
```

Comment:
This was necessary because otherwise my panel would not load. web.php does not exist then I get as error